kinetic/distribution.yaml

```diff
@@ -3805,6 +3805,21 @@ repositories:
       url: https://github.com/ros-drivers/linux_peripheral_interfaces.git
       version: kinetic
     status: maintained
+  lkh:
+    release:
+      packages:
+      - glkh_solver
+      - lkh
+      - lkh_solver
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/crigroup/lkh-release.git
+      version: 0.1.1-0
+    source:
+      type: git
+      url: https://github.com/crigroup/lkh.git
+      version: 0.1.0
```

Comment:
The source and release versions don't match, is it on purpose that this is tracking an older version for the source than the release ? Most people track their default branch ( in this case "master") for their source entry. This will allow you to leverage automatic CI jobs for each new commit on your master branch

Reply:
My bad. Thanks for the pointer